### PR TITLE
CounterTack new commands

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/15990

## Does it break backward compatibility?
   - No

## Must have
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Integrations/CounterTack/CounterTack.yml [hunk 1]
```diff
@@ -0,0 +1,1296 @@
...
+  type: 0
+- display: User Name
+  name: credentials
+  required: true
+  type: 9
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Trust any certificate (unsecure)
```
Comment (Itay4) [inline Integrations/CounterTack/CounterTack.yml]: please change `unsecure` to `not secure`

File: Integrations/CounterTack/CounterTack.yml [hunk 2]
```diff
@@ -0,0 +1,1296 @@
...
+      type: String
+    - contextPath: Endpoint.OSVersion
+      description: The endpoint sensor version.
+      type: String
+    - contextPath: Endpoint.Id
+      description: The ID of the Endpoints.
+      type: String
+  - arguments:
+    - default: false
+      description: ' The CQL expression to be used for the search(e.g behaviors.event_count<60)'
```
Comment (Itay4) [inline Integrations/CounterTack/CounterTack.yml]: remove whitespace

File: Integrations/CounterTack/CounterTack.yml [hunk 3]
```diff
@@ -0,0 +1,1296 @@
...
+      description: The time that the behavior was last active.
+      type: Date
+    - contextPath: CounterTack.Behavior.ImpactLevel
+      description: The threat level of the behavior (LOW, MEDIUM, HIGH, CRITICAL).
+      type: String
+  isfetch: true
+  runonce: false
+  script: '-'
+  type: python
+releaseNotes: New commands
```
Comment (Itay4) [inline Integrations/CounterTack/CounterTack.yml]: improve release notes with command names

File: Integrations/CounterTack/CounterTack.py [hunk 4]
```diff
@@ -0,0 +1,1411 @@
...
+    context = {
+        'CounterTack.Behavior(val.Id && val.Id === obj.Id)': createContext(behaviors, keyTransform=underscoreToCamelCase)
+    }
+    headers = ['Name', 'Id', 'Type', 'Max_impact', 'Impact_Level', 'Endpoint_ID', 'lastReported']
+    entry = {
+        'Type': entryTypes['note'],
+        'Contents': behaviors,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('CounterTack Endpoints Behaviors:', data, headers, removeNull=True),
```
Comment (Itay4) [inline Integrations/CounterTack/CounterTack.py]: remove `:`

File: Integrations/CounterTack/CounterTack.py [hunk 5]
```diff
@@ -0,0 +1,1411 @@
...
+        data.append({
+            'Id': behavior.get('id'),
+            'Name': behavior.get('name'),
+            'Type': behavior.get('type'),
+            'Impact_Level': behavior.get('impact_level'),
+            'lastReported': behavior.get('last_reported'),
+            'Endpoint_ID': behavior.get('endpoint_id')
+        })
+
+    context = {
```
Comment (Itay4) [inline Integrations/CounterTack/CounterTack.py]: the context does not match in 100% to the `get-behavior` command, e.g. `EndpointID != EndpointId`
please make sure they fit

File: Integrations/CounterTack/CounterTack.py [hunk 6]
```diff
@@ -0,0 +1,1411 @@
...
+        'CounterTack.Behavior(val.Id && val.Id === obj.Id)': createContext(behaviors, keyTransform=underscoreToCamelCase)
+    }
+    headers = ['Name', 'Id', 'Type', 'Max_impact', 'Impact_Level', 'Endpoint_ID', 'lastReported']
```
Comment (Itay4) [inline Integrations/CounterTack/CounterTack.py]: headers should not contain underscores

File: Integrations/CounterTack/CounterTack.py [hunk 7]
```diff
@@ -0,0 +1,1411 @@
...
+
+
+def get_endpoints():
+
+    """
+    Returns the information on existing endpoints
+    """
+    data = []
+    endpoint_standards = []
+    context = {}
```
Comment (Itay4) [inline Integrations/CounterTack/CounterTack.py]: not need to initialise it here

File: Integrations/CounterTack/CounterTack.py [hunk 8]
```diff
@@ -0,0 +1,1411 @@
...
+            'Domain': endpoint.get('domain'),
+            'MACAddress': endpoint.get('mac'),
+            'OS': endpoint.get('product_name'),
+            'OSVersion': endpoint.get('driver_version'),
+            'Model': endpoint.get('current_profile'),
+            'Memory': endpoint.get('memory'),
+            'Processors': endpoint.get('num_cpus')
+        })
+
+    context['CounterTack.Endpoint(val.Id && val.Id === obj.Id)'] = createContext(endpoints, keyTransform=underscoreToCamelCase)
```
Comment (Itay4) [inline Integrations/CounterTack/CounterTack.py]: build the whole context dict here, e.g. 
```
context = {
     'CounterTack.Endpoint(val.Id && val.Id === obj.Id)': ...,
     'Endpoint': ....
}
```

**Comment by Shellyber** [comment]
Hey Please review the changes

**Pull request merged**